theme/static/js/collaborate.js

```diff
@@ -107,6 +116,30 @@ $(document).ready(function () {
 
 });
 
+/**
+ * hide or show search feedback message
+ * @param visible
+ */
+function set_feedback_visible(visible = false) {
```

Comment:
I think this method is unnecessary since we can use the `show` and `hide` methods directly on the selection:
```
$("#id-Group-Search-Result-Msg").hide();
$("#id-Group-Search-Result-Msg").show();
```